sessions/nextjs-basics-and-routing/lotr-app/README.md

```diff
@@ -0,0 +1,81 @@
+# Next.js Basics: Lord of the Rings App
+
+You are going to create a fan page about the Lord of the Rings books.
+
+## Create the App
+
+First create a new Next.js based on our template using this command:
+
+```
+npx ghcd@latest neuefische/web-exercises/tree/main/templates/next lotr-app -i
+```
+
+> 💡 Our Next.js template is very similar to `create-next-app` but it also installs additional dependencies. It adds support for testing using Jest and also installs Storybook and Styled Components, which you will learn about in the upcoming sessions.
+
+Download and extract the [Resources](#resources) into your project folder.
+
+## Create the pages
+
+Create the following pages:
+
+| Route                                 | Purpose                                                |
+| ------------------------------------- | ------------------------------------------------------ |
+| `/`                                   | Display link to the volume overview `/volumes`         |
+| `/volumes`                            | Display list with links to all volumes                 |
+| `/volumes/the-fellowship-of-the-ring` | Display data for volume 1 (The Fellowship of the Ring) |
+| `/volumes/the-two-towers`             | Display data for volume 2 (The Two Towers)             |
+| `/volumes/the-return-of-the-king`     | Display data for volume 3 (The Return of the King)     |
+
+> 💡 A quick heads up about The Lord of the Rings: The book series has three volumes: The Fellowship of the Ring, The Two Towers and The Return of the King. Each volume contains two books for a total of six books.
+
+## Overview Page Contents
+
+> ❗️ **Ignore all styling for now.** Just use the default browser styles. You will add styling in an upcoming session.
+
+The overview page should contain the following:
+
+- A heading with the title "Lord of the Rings" (`h1`)
+- The introduction text imported from `lib/data.js` (`p`)
+  ```js
+  import { introduction } from "../lib/data";
```

Comment:
Since we are inside of `pages/volumes/index.js`, we need to go one step further up:
```suggestion
  import { introduction } from "../../lib/data";
```